go.mod

```diff
@@ -30,6 +28,10 @@ require (
 	golang.org/x/term v0.0.0-20210422114643-f5beecf764ed // indirect
 	golang.org/x/tools v0.1.7
 	gopkg.in/gormigrate.v1 v1.6.0
```

Comment:
can you check why we still have this dep here?

Reply:
Resolved in 9cee4e8